### DISCUSSION TITLE
Missing some triage permissions on iOS

### DISCUSSION THREAD

**Discussion by lucascolley** [community/community#95963] [Mobile]
I see errors like ‘{user} does not have the correct permissions to execute `UpdatePullRequest`’ on the iOS app when trying to request reviews and modify milestones on a repo I have triage rights on. I can do these things on browsers but not through the app on iOS. The app seems to be limited to just adding labels and closing issues/PRs.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.145.0 (120653769)
Device: iPhone 11 Pro (iOS 17.2.1)
</details>

**Comment by eliperkins** [comment]
Hey @lucascolley, thanks for this feedback! I can understand why you would be frustrated if the experience and permissions for triaging repos on GitHub Mobile versus the web browser were so different. This sounds like a bug on our end, so I've created an issue on our internal tracking repo to dig in more. I'll follow up with the engineering team and let you know when this has been addressed. Thanks for bringing this to our attention!

**Reply by lucascolley** [reply]
thanks @eliperkins ! happy to provide more info if that helps.
